### PR TITLE
Update gce initialization scripts

### PR DESCRIPTION
**Pull request by Georgehe4**
The script updates will
1) Remove un-necessary steps for storing data onto GCS (shared buckets).
2) Automatically install maven into master.
3) Automatically update node and npm in master.

### CODE REVIEW

File: bin/gce/install.sh [hunk 1]
```diff
@@ -55,12 +55,16 @@ if [[ "${ROLE}" == 'Master' ]]; then
...
-    # Install google cloud nio 
-    curl -L https://oss.sonatype.org/content/repositories/releases/com/google/cloud/google-cloud-nio/0.22.0-alpha/google-cloud-nio-0.22.0-alpha-shaded.jar  | gsutil cp - gs://mango-initialization-bucket/google-cloud-nio-0.22.0-alpha-shaded.jar
     
+    # update node and npm version using Debian, as root
+    curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
+    apt-get -y install nodejs
+    npm install -g n
+
 fi
 echo "Completed installing Jupyter!"
```
Comment (akmorrow13) [inline bin/gce/install.sh]: Is this installation file just for installing Jupyter?

File: bin/gce/install.sh [hunk 2]
```diff
@@ -55,12 +55,16 @@ if [[ "${ROLE}" == 'Master' ]]; then
        stable"
```
Comment (akmorrow13) [inline bin/gce/install.sh]: Can you add a comment at the top of this script summarizing everything this script is installing?

File: bin/gce/install.sh [hunk 3]
```diff
@@ -57,16 +72,19 @@ if [[ "${ROLE}" == 'Master' ]]; then
...
     apt-get -y install nodejs
+    echo "Completed installing nodejs!"
     npm install -g n
+    echo "Completed installing npm!"
 
 fi
-echo "Completed installing Jupyter!"
 
 # Install Jupyter extensions (if desired)
 # TODO: document this in readme
```
Comment (akmorrow13) [inline bin/gce/install.sh]: What do we want to document in the README?

File: bin/gce/install.sh [hunk 4]
```diff
@@ -1,5 +1,17 @@
...
+#   jupyter
+#   npm
+#   nodejs
+#   maven
+#   docker
+#   google_compute_engine [pip]
+#   cigar [pip]
+# Installs the following on dataproc workers:
+#   conda
+#   cigar [pip]
```
Comment (akmorrow13) [inline bin/gce/install.sh]: Im still unsure of why we need to install cigar. shouldn't this be installed when we install bdgenomics.mango?

Comment (Georgehe4) [reply bin/gce/install.sh]: We'll probably have to distribute it in the PYSPARK_PYTHON to workers -- it looks like we currently don't do it. I think we should address this in a follow-up change.

https://www.cloudera.com/documentation/enterprise/5-5-x/topics/spark_python.html

**Comment by akmorrow13** [comment]
@Georgehe4 sorry I dropped this, is this ready for merge?

**Pull request merged**